src/OpenLoco/CMakeLists.txt

```diff
@@ -811,6 +811,9 @@ else ()
         PRIVATE
             DEBUG=${DEBUG_LEVEL}
             _NO_LOCO_WIN32_=1)
+
+    target_link_libraries(OpenLoco
+        ZLIB::ZLIB) # This shouldn't be required but vcpkg seems to not map the dependencies correctly for png static
```

Comment:
This is also in the MSVC section above, but is not an MSVC issue but a static linking one. Ideally both places should be refactored out to a `if(STATIC_LINKING)` or something but I couldn't work out how to do that.

I strongly suspect I'll have broken the build by doing this indiscriminately here, wondering how to do this properly?

Reply:
I would just move it up to L774 and remove it from the win32 branch. Its not like it matters much if we link this in even if we aren't using it directly atm.

Reply:
Sounds good, I've pushed this up now, looks a lot cleaner